lang/strings_english.txt

```diff
@@ -869,6 +869,7 @@ $s_category_deleted_msg = 'Category has been successfully deleted...';
 
 # manage_proj_category_delete_page.php
 $s_category_delete_sure_msg = 'Are you sure you want to delete this category?<br />Category "%1$s" contains %2$d issues.';
+$s_category_delete_confirm_msg = 'Are you sure you want to delete this category?<br />Category "%1$s" ';
```

Comment:
I would use the following string instead:
'Are you sure you want to delete "%1$s" category?'